scholia/app/templates/venue.html

```diff
@@ -39,7 +39,7 @@
 
  prolificAuthorsSparql = `
 # Prolific authors for a specific journal
-SELECT ?count ?author ?authorLabel ?orcid
+SELECT ?count ?author ?authorLabel ?ORCID
```

Comment:
Yes, change it back for now. We should fix the header at some point. There is also the problem of generated a link when the cell is empty